scripts/copyright.sh

```diff
@@ -44,7 +44,7 @@ fi
 if command -v nwa &> /dev/null; then
   nwa add -l apache -c "$AUTHOR" src dist tests scripts
 elif command -v docker &> /dev/null; then
-  docker run -it -v "${PWD}:/src" ghcr.io/b1nary-gr0up/nwa:main add -l apache -c "$AUTHOR" src dist tests scripts
+  docker run --rm -v "${PWD}:/src" ghcr.io/b1nary-gr0up/nwa:main add -l apache -c "$AUTHOR" src dist tests scripts
```

Comment:
This is because `-it` is "interactive" mode and tries to attach to your terminal, which does not work in husky pre-commit script:
```
the input device is not a TTY
husky - pre-commit script failed (code 1)
```

Also if you don't include `--rm`, it will leave an orphaned container on your disk every time (=on every commit).